implementations/rust/ockam/ockam_command/src/util/duration.rs

```diff
@@ -0,0 +1,59 @@
+use clap::error::{Error, ErrorKind};
+use lazy_static::lazy_static;
+use regex::Regex;
+use std::time::Duration;
+
+lazy_static! {
+    static ref DURATION_REGEX: Regex =
+        Regex::new(r"(?P<numeric_duration>[0-9]+)(?P<length_sigil>m|s|ms)?$").unwrap();
+}
```

Comment:
We use `once_cell`, which is less verbose. Check [this out as an example](https://github.com/build-trust/ockam/blob/13f093392ea1e2a8a44dad71870996b5fdc746b3/implementations/rust/ockam/ockam_command/src/run/parser.rs#L293).

Reply:
Now that I think about it, [`OnceCell` is already part of the std library](https://doc.rust-lang.org/stable/std/cell/struct.OnceCell.html) since 1.70, which we support on our CI pipeline, so feel free to use that version instead.

Reply:
Sounds good.  I've switched it to use `std::sync::OnceLock`, since statics need to be thread-safe.

Reply:
We can't use the `std` types yet, as CI is using rust's 1.68 version. There is a [PR in progress](https://github.com/build-trust/ockam/pull/5514) to upgrade it to the latest version, but I'd recommend you to use once_cell in the meantime. We'll do the switch to std after that PR gets merged.